src/Service/ServiceUtils.cpp

```diff
@@ -391,20 +391,21 @@ static ErrorMessageOr<fs::path> FindSymbolsFilePathElf(
 }
 
 ErrorMessageOr<fs::path> FindSymbolsFilePath(
-    const fs::path& module_path, const std::vector<fs::path>& search_directories) noexcept {
-  auto object_file_or_error = CreateObjectFile(module_path);
+    const orbit_grpc_protos::GetDebugInfoFileRequest& request,
+    const std::vector<fs::path>& search_directories) noexcept {
+  auto object_file_or_error = CreateObjectFile(request.module_path());
```

Comment:
Nit: From an API design perspective this function shouldn't get the request proto. Right now it's kind of okay because the request proto only contains fields that are actually needed by this function. But you will put a burden on the next person adding an unrelated field to the request proto which makes it not suitable anymore as an input parameter type to this function.

Reply:
I decided for the trade off to not change this now and leave the burden on the next person.